fastapi_etebase/routers/authentication.py

```diff
@@ -0,0 +1,177 @@
+from datetime import datetime
+from functools import cached_property
```

Comment:
Now that I think about this, this makes us Python 3.8+ which I don't think is a good idea. We probably wanna be 3.7+ if possible.